src/core/Akka.Tests/Event/EventStreamSpec.cs

```diff
@@ -89,18 +90,18 @@ public void Not_allow_null_as_subscriber()
         public void Not_allow_null_as_unsubscriber()
         {
             var bus = new EventStream(true);
-            XAssert.Throws<ArgumentNullException>(() =>
+            Assert.Throws<ArgumentNullException>(() =>
```

Comment:
If you're changing XAssert to Assert, you might as well do it to all of it

Reply:
Okay